Update `staking-deposit-cli` to `ethstaker-deposit-cli`

chong-he opened the pull request:
Based on the feedback on [Discord](https://discord.com/channels/605577013327167508/605577013331361793/1375339231970328657), we should update `staking-deposit-cli` to `ethstaker-deposit-cli` as the former has not been updated for a while (e.g., not supporting Hoodi). 

Ethstaker is now maintaining the deposit cli: https://www.reddit.com/r/ethstaker/comments/1ecuj5q/the_depositcli_has_long_needed_some_critical/

~There are some files like `validator_manager/src/common.rs` and `validator_manager/test_vectors/generate.py` still~ ~refering to `staking-deposit-cli`.~

~The `common.rs` is referring to this commit:~
~https://github.com/ethereum/staking-deposit-cli/blob/76ed78224fdfe3daca788d12442b3d1a37978296/~staking_deposit/credentials.py#L131-L144~
~which is changed in `ethstaker-deposit-cli`:~
~https://github.com/eth-educators/ethstaker-deposit-cli/blob/main/ethstaker_deposit/credentials.py~

~So I leave it untouched for now~
Edit: updated all to `ethstaker-deposit-cli` 

Other updates:
- added a note on Homebrew page as Lighthouse on Homebrew hasn't been updated since v7.0.0. [Review here](https://github.com/sigp/lighthouse/pull/7518/files#diff-fd14e06257fea0886d9281f1d8e833d40460103bb4406baf77ce2cd29ed2a520R8)
- added `--vc-url` in validator manager doc as per: https://discord.com/channels/605577013327167508/605577013331361793/1377638933579436032. [Review here](https://github.com/sigp/lighthouse/pull/7518/files#diff-0467838cad89c67f549e4e4e6b8c8c180eafb6b648b1e40001bb3c692c8174dcR3)

chong-he committed:
update to ethstaker-deposit-cli

chong-he committed:
minor

chong-he committed:
minor

chong-he committed:
update Homebrew

chong-he committed:
minor

jimmygchen commented on book/src/mainnet_validator.md:
```diff
@@ -42,7 +42,7 @@ hardware. 32 ETH is a significant outlay and joining a testnet is a great way to
 ### Step 1. Create validator keys
 
-The Ethereum Foundation provides the [staking-deposit-cli](https://github.com/ethereum/staking-deposit-cli/releases) for creating validator keys. Download and run the `staking-deposit-cli` with the command:
+The Ethereum Foundation provides the [ethstaker-deposit-cli](https://github.com/eth-educators/ethstaker-deposit-cli/releases) for creating validator keys. Download and run the `ethstaker-deposit-cli` with the command:
```
```suggestion
EthStaker provides the [ethstaker-deposit-cli](https://github.com/eth-educators/ethstaker-deposit-cli/releases) for creating validator keys. Download and run the `ethstaker-deposit-cli` with the command:
```

chong-he replied:
Thanks, nice catch!

jimmygchen reviewed (changes requested):
There's another reference here:
https://github.com/sigp/lighthouse/blob/a2b00090fd8c6c5d6b4ffc88f8d0f937d9165c58/account_manager/src/validator/import.rs#L35

and here
https://github.com/sigp/lighthouse/blob/d60c24ef1cc0b5dfa930e1dd4fc85abc29e5fc4c/common/account_utils/src/validator_definitions.rs#L453-L457

chong-he committed:
Update book/src/mainnet_validator.md

Co-authored-by: Jimmy Chen <jchen.tc@gmail.com>

chong-he committed:
Add vc-url in vm

chong-he committed:
update

chong-he committed:
spell check

chong-he committed:
update all

chong-he committed:
Merge branch 'unstable' into book-deposit-cli

chong-he commented:
> There's another reference here:
> 
> https://github.com/sigp/lighthouse/blob/a2b00090fd8c6c5d6b4ffc88f8d0f937d9165c58/account_manager/src/validator/import.rs#L35
> 
> and here
> 
> https://github.com/sigp/lighthouse/blob/d60c24ef1cc0b5dfa930e1dd4fc85abc29e5fc4c/common/account_utils/src/validator_definitions.rs#L453-L457

Updated, thanks!

Also updated `validator_manager/src/common.rs` and `validator_manager/test_vectors/generate.py`

jimmygchen reviewed (approved):
LGTM!
